src/3p-frame.js

```diff
@@ -109,8 +106,7 @@ export function getIframe(parentWindow, parentElement, opt_type, opt_context) {
     // Chrome does not reflect the iframe readystate.
     this.readyState = 'complete';
   };
-  iframe.setAttribute('data-amp-3p-sentinel', attributes._context[
-    sentinelNameChange ? 'sentinel' : 'amp3pSentinel']);
+  iframe.setAttribute('data-amp-3p-sentinel', attributes._context['sentinel']);
```

Comment:
Wait, we are changing the sentinel name. Shouldn't we do `iframe.setAttribute('data-sentinel`) here.